docs/src/use/configure/migration-guide.md

```diff
@@ -255,6 +255,63 @@ export default [
 ];
 ```
 
```

Comment:
Should there be a new heading here like ``### `eslint-env` Configuration Comments``?

Reply:
Yes, I'd add a new heading.